### DISCUSSION TITLE
📣 Epic Stack Users

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#557] [General]
If you're using the Epic Stack (or even if you find yourself referencing and implementing pieces of it in your existing app a lot), please let me know here! I have some collaboration opportunities I want to visit with you about.

If you have a link to the deployed version and/or source code that would be very helpful, but not required (feel free to just describe how you've used it).

Thanks a lot!

**Comment by alexanderson1993** [comment]
Hey Kent! I reference Epic Stack regularly and used it as the basis for the svg-icons-cli package.

**Comment by ck9891** [comment]
Hey Kent, actively developing locally on a project that uses the Epic Stack.

**Comment by luisiacc** [comment]
I'm using the epic stack on one of my side-projects and is generating actual revenue, it's on https://swapmyface.app . I also incorporated a blog, pretty much ripped off part of your logic from https://kentcdodds.com and integrated it in the stack. I also started another project using it but it's not ready yet, but it will be basically templates for lawyers and a lot of other functionalities on top of that.

**Comment by cr101** [comment]
I've just started building a web app similar to PitchBook/Crunchbase using Epic Stack and Drizzle ORM. Thanks Kent!

**Comment by michaelschwobe** [comment]
Thanks Kent! The epic-stack has helped tremendously with learning and building my own ideas. Here's my current WIP: https://github.com/michaelschwobe/tagsfordays

**Comment by theskillwithin** [comment]
Using parts of the Epic Stack for https://bestofai.com/

**Comment by sys13** [comment]
I'm using it in https://www.clientcruise.com. I am currently doing the heavy lift of porting to drizzle and postgres.

**Reply by skl3** [reply]
Do you have a fork with the port to drizzle and postgres available? Would be super helpful for a project I'm bootstrapping.

**Reply by nichtsam** [reply]
Hey @skl3, not sure if this will help but my project uses `Drizzle`.
Not exactly using `epic-stack` but I tries to implement the same practices.

Check the comment below: https://github.com/epicweb-dev/epic-stack/discussions/557#discussioncomment-7880142

**Comment by dawnmist** [comment]
I have been using it heavily as a reference. Staging deployment is at https://staging.supportbudget.com (currently has the various integration & testing bits set up, but the bulk of the app itself hasn't been implemented yet).

**Comment by Varkoff** [comment]
Hi Kent ! We're using parts of the Epic Stack for https://goodcollect.co

**Comment by luksch42** [comment]
We're using parts of it as a reference to build https://roastrepublic.co

**Comment by petar-i-todorov** [comment]
I've used it recently on a hackathon project for university. Since it was a group project, we didn't win because we lacked in other directions (such as presentation, etc.) but everyone (including the professors) was amazed that I was able to build that much alone in just 24 hrs.

Also, I'm referencing it currently to build my portfolio project. Will add a link once I deploy it.

**Comment by samialdury** [comment]
Hi, I'm using the Epic Stack for multiple side projects which are not public yet, but I'll link them here when they are. 
I replaced SQLite with MySQL ([PlanetScale](https://planetscale.com/)) and deployment is done to multiple VPSs via [Pulumi](https://www.pulumi.com/) and [Kamal](https://kamal-deploy.org/). 
Thank you for maintaining it!

**Comment by clovis1122** [comment]
Hi Kent, thank you for your work. I'm using a modified version of the epic stack for a few projects:

* An app I'm developing with some friends for selling car parts (the project is still a WIP but hosted at https://wiparts.vercel.app/).
* An app at work to generate candidate pitches: https://app.candidatepitch.com/login. The root domain is using Wix. I changed almost everything infrastructure-related, borrowing heavily from the [grunge-stack](https://github.com/remix-run/grunge-stack). But the base is from the epic-stack.
* Migrating two sites https://jobs.recruiter.com and https://www.mediabistro.com/jobs from Rails to Remix based on the epic-stack. Not all the pages are on remix yet. This is the project I had to customize the most from the original epic-stack as it was an existing project, so I made changes from infrastructure, database (DrizzleORM FTW!), and even session storage (to make it compatible with our existing Rails site).

**Comment by kentcdodds** [comment]
Thanks to everyone who has posted your projects! I'm announcing something in January and I'll reach out to you all about things more by then. Cheers!

**Comment by Mr-Baker-JPG** [comment]
I've used it as the base of an app that us used at our school for some in house projects (lunch ordering, SMS messaging and time off calendaring).  It is integrated with our SIS.
I've also used to to develop a few small websites for friends.

**Comment by devlinmorrow** [comment]
Hiya! I'm using it in a language-learning web app that I hope to release one day soon :)

**Comment by jordisantamaria** [comment]
Hi, at nyxidiom we are launching today pro.miistico.com, created with the epic stack.

**Comment by fdietze** [comment]
After some evaluation, we decided to stick with epic-stack for our prototype. We're building a social network with a novel attention algorithm: https://github.com/social-protocols/social-network

**Comment by yushanwebdev** [comment]
Currently, I'm using that as a reference point. While going through the Epic Web materials, I'm learning about some of its utilities. However, once I complete the Epic Web course, I plan to heavily use the Epic Stack.

**Comment by nichtsam** [comment]
My personal website. 
I basically go through `epic-stack`'s docs, reads the decisions, and try to implement them myself, referencing `epic-stack`'s implementations.

Some differences:
- I use `Turso` instead of `Litefs`
- I use `Drizzle` instead of` Prisma`
- I use `Pnpm` instead of `Npm`

https://nichtsam.com/
https://github.com/nichtsam/nichtsam.com

Really grateful for such a great stack.

**Comment by skorfmann** [comment]
Explored it over the last few days and gonna dive deeper. Helped me tremendously to put the pieces together. Gonna revamp https://www.hekto.co/ based on this

**Comment by felixyeboah** [comment]
Building my 5th app with Epic Stack and I couldn't be happier. previous build https://dnaweds.love/ and currently building this https://misscookie-4e2a-staging.fly.dev/ with Epic Stack.

**Comment by jansedlon** [comment]
Hi, our app for [flixy.cz](flixy.cz) is initially based on epic-stack. Different deployment, etc.. but boilerplate was made with epic stack

**Comment by DennisKraaijeveld** [comment]
Hi @kentcdodds I am building [Spedix.io](https://spedix.io/) with Epic Stack as starter. All basics are still there like the dark/light mode, authentication and deployment but many things have been built on top of it. 

- CRM
- (Marketing) Automations
- Emails (builder)
- Website generations
- Real time notifications
- Embedded CMS
- Website analytics (Using Tinybird)

All is done by myself, while I am having a full time job as well. But the Epic Stack foundations are so strong, that it helped me developing a really strong app in a short time
